web/src/info_overlay.js

```diff
@@ -150,6 +166,23 @@ Coffee`,
     </li>
     </ul>
 </div>
+`,
+    },
+    {
+        markdown: "/todo",
+        output_html: `\
+<div class="message_content rendered_markdown">
+   <div class="widget-content">
+      <div class="todo-widget">
+         <h4>Task list</h4>
+         <div class="add-task-bar">
+            <input type="text" class="add-task todo-task-spacer" placeholder="New task">
+            <input type="text" class="add-desc todo-task-spacer" placeholder="Description">
+            <button class="add-task">Add task</button>
+         </div>
+      </div>
+   </div>
+</div>
```

Comment:
So I think we want the `/todo` example to be one where we pass more details when making it, like we do the poll, so that you can see what it will look like.

Reply:
Yeah, that makes sense. I have updated the code to now include two todos; one of them has been marked as checked and hence is crossed over.

![image](https://github.com/zulip/zulip/assets/35286603/5bb583ab-5020-4153-ba98-d8511c7642da)

The only minor issue is that ticking/unticking the checkboxes has no effect on the task.

Reply:
Yeah that's probably OK.

Reply:
Makes sense.